spec/unit/base64_attach_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
```

Comment:
I'm more inclined to put this under a `lib` folder instead of unit. What do the others think?

Reply:
Done